crates/tako/src/scheduler/state.rs

```diff
@@ -83,8 +92,20 @@ impl SchedulerState {
         task: &Task,
         taskmap: &TaskMap,
         worker_map: &Map<WorkerId, Worker>,
+        try_stick: bool,
```

Comment:
Please document this parameter, it's far from obvious.

Reply:
Renamed and removed forgotten invalid comment